src/Microsoft.FeatureManagement/Telemetry/AssignmentReason.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+namespace Microsoft.FeatureManagement.Telemetry
+{
+    /// <summary>
+    /// The reason the variant was assigned during the evaluation of a feature.
+    /// </summary>
+    public enum AssignmentReason
+    {
+        /// <summary>
+        /// No variant is assigned.
+        /// </summary>
+        None,
+
+        /// <summary>
+        /// Variant is assigned by default after processing the user/group/percentile allocation, when the feature flag is disabled.
```

Comment:
When the feature flag is disabled, we don't process "the user/group/percentile allocation".